packages/glibc/0003-nscd-Do-not-rebuild-getaddrinfo-bug-30709.patch

```diff
@@ -0,0 +1,185 @@
+From 6b99458d197ab779ebb6ff632c168e2cbfa4f543 Mon Sep 17 00:00:00 2001
+From: Florian Weimer <fweimer@redhat.com>
+Date: Fri, 11 Aug 2023 10:10:16 +0200
+Subject: [PATCH 3/6] nscd: Do not rebuild getaddrinfo (bug 30709)
+
+The nscd daemon caches hosts data from NSS modules verbatim, without
```

Comment:
If we are not building with `nscd`, do we still need this patch?

Reply:
It's easier to maintain the patch series if we just take the recent upstream commits - per spec file instructions - unless there's something of specific concern.